### DISCUSSION TITLE
coreemu cannot be used with docker?

### DISCUSSION THREAD

**Discussion by boyan7777** [coreemu/core#834] [Q&A]
When I use docker node to connect to rj45, I can't seem to get the ipaddress. Even if I remove these nodes and then use pc node to connect to RJ45, I still can't get the IP address unless I restart UBUNTU.

I found that as long as I execute docker run on the command line, the coreemu network will fail.
May I ask what I may have done wrong?

os:Ubuntu 22.04.3 LTS
coreemu:CORE daemon v.9.0.3
docker:Docker version 24.0.7

**Comment by bharnden** [comment]
I need a clearer picture of your issue, I don't quite understand what you are explaining. You will have to be very detailed and descriptive.

**Comment by boyan7777** [comment]
![img](https://github.com/coreemu/core/assets/150875680/5c6c301e-bcee-467f-b8c6-23f790905f43)


pic 1 : ubuntu interface
pic 2 : docker node config
pic 3 : run docker connect with rj45 

pic4 : pc node connect with rj45 and work fine

Why is the docker node network blocked?
